### PR TITLE
bump version for api-mapper

### PR DESCRIPTION
**Pull request by odra**

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by odra** [comment]
[deploy][lrossett]

File: global.json [hunk 1]
```diff
@@ -1134,11 +1134,11 @@
         "id": "api-mapper-service",
         "name": "API Mapper",
         "type": "cloud_nodejs",
-        "repoUrl": "git://github.com/feedhenry-templates/fh-api-mapper.git",
-        "githubUrl": "git://github.com/feedhenry-templates/fh-api-mapper.git",
-        "repoBranch": "refs/heads/master",
+        "repoUrl": "git://github.com/odra/fh-api-mapper.git",
+        "githubUrl": "git://github.com/odra/fh-api-mapper.git",
```
Comment (secondsun) [inline global.json]: Probably shouldn't be changing the url.

File: global.json [hunk 2]
```diff
@@ -1125,7 +1125,7 @@
       "name": "API Mapper",
       "priority": 0.883,
       "id": "rest-client",
-      "docs": "https://raw.githubusercontent.com/feedhenry-templates/fh-api-mapper/master/README.md",
+      "docs": "https://raw.githubusercontent.com/odra/fh-api-mapper/RHMAP-6206-fix/README.md",
```
Comment (secondsun) [inline global.json]: Probably shouldn't be changing the url.

Comment (matzew) [inline global.json]: 👍

Comment (matzew) [inline global.json]: I think this was done for testing, on the cluster, no @odra ?

Comment (odra) [inline global.json]: Correct, once tested I will revert it to use the default url.

Comment (secondsun) [inline global.json]: Why does the URL need to be changed for testing?  Ideally that PR will be tested and merged in isolation and then this one would be tested against master.

**Comment by odra** [comment]
[deploy][lrossett]

**Comment by fh-automatron** [comment]
Starting a deploy to [lrossett] - check the status checks in this fhcap PR for more details: https://github.com/fheng/fhcap/pull/2590

**Comment by secondsun** [comment]
@odra What was in this fix?  I'm trying to get the saml-xamarin-app into fhcap master and FH-v3.12 but this PR has jumped in front and now I don't know what is in my PRs to fhcap.

**Comment by odra** [comment]
@secondsun it was a fix needed for RHMAP-6206, just a version bump because of the changes on fh-api-mapper.

**Comment by secondsun** [comment]
But what was in the change?

Summers Pittman
>>Phone:404 941 4698
>>Java is my crack.


On Tue, Jul 12, 2016 at 8:04 PM, Leonardo Rossetti <notifications@github.com

**Comment by odra** [comment]
A change on views/upgrade.html: [#69](https://github.com/feedhenry-templates/fh-api-mapper/pull/69/files)

**Comment by matzew** [comment]
@secondsun to our understanding the version bump is needed, just once - I guess yours should be in than too 😄 

yeah, it's not optimal / best solution

**Pull request merged**